api-catalog-package/src/main/resources/bin/start.sh

```diff
@@ -119,17 +119,10 @@ LIBPATH="$LIBPATH":"${LIBRARY_PATH}"
 
 keystore_type="${ZWE_configs_certificate_keystore_type:-${ZWE_zowe_certificate_keystore_type:-PKCS12}}"
 keystore_pass="${ZWE_configs_certificate_keystore_password:-${ZWE_zowe_certificate_keystore_password}}"
+key_pass="${ZWE_configs_certificate_key_password:-${ZWE_zowe_certificate_key_password:-${keystore_pass}}}"
 truststore_type="${ZWE_configs_certificate_truststore_type:-${ZWE_zowe_certificate_truststore_type:-PKCS12}}"
 truststore_pass="${ZWE_configs_certificate_truststore_password:-${ZWE_zowe_certificate_truststore_password}}"
 
-# There was an issue where java could throw an exception on keyring read if the password was blank
```

Comment:
I'm not totally sure about this removal. Wouldn't be failing anyway if the fallback (`keystore_pass`) In the `key_pass="${ZWE_configs_certificate_key_password:-${ZWE_zowe_certificate_key_password:-${keystore_pass}}}` won't be defined?

Reply:
the removal was a proposed workaround for JCERACF keyrings, where "dummy" password seemed to be accepted, this assumption turned out incorrect and didn't work on a few systems. therefore it is better to let users set "dummy" as a password if that would work for them but do not set this for everyone without an option to use an empty password

Reply:
Ok I undestand, thanks